action-api/src/main/java/com/powsybl/action/AreaInterchangeTargetUpdateActionBuilder.java

```diff
@@ -0,0 +1,49 @@
+package com.powsybl.action;
+
+public class AreaInterchangeTargetUpdateActionBuilder implements ActionBuilder<AreaInterchangeTargetUpdateActionBuilder> {
+
+    private String id;
+
+    private String areaId;
+
+    private Double target = null;
+
+    @Override
+    public String getType() {
+        return AreaInterchangeTargetUpdateAction.NAME;
+    }
+
+    @Override
+    public AreaInterchangeTargetUpdateActionBuilder withId(String id) {
+        this.id = id;
+        return this;
+    }
+
+    @Override
+    public String getId() {
+        return id;
+    }
+
+    @Override
+    public AreaInterchangeTargetUpdateActionBuilder withNetworkElementId(String elementId) {
+        return this;
+    }
+
+    public AreaInterchangeTargetUpdateActionBuilder withTarget(Double target) {
```

Comment:
primitive double here

```suggestion
    public AreaInterchangeTargetUpdateActionBuilder withTarget(double target) {
```